src/test/resources/better_errors/weirdfunct.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
what is this file testing?

Reply:
I think this was leftover from some experiment, I've removed it for now.